src/components/Item/ItemPreview.js

```diff
@@ -104,7 +104,12 @@ const ItemPreview = ({
         <Button size="small" color="primary" onClick={() => routeToItem(_id)}>
           Details
         </Button>
-        <Rating isReadOnly={true} />
+        <Rating
```

Comment:
let's just remove the entire rating on the item preview

Reply:
done